app/templates/brief.html

```diff
@@ -138,16 +138,13 @@ <h1>{{ brief.title }}</h1>
     </div>
   </div>
 {% elif brief.status == 'live' %}
+  <br/><!-- <- remove once grid and body elements have been converted to govuk frontend -->
```

Comment:
The new styles will sort it out, yes. On my preview feature, adding `<p class="govuk-body">` for the secondary-action-link (above the button) did the trick.